recipe/build_base.sh

```diff
@@ -450,11 +496,9 @@ pushd "${PREFIX}"/lib/python${VER}
   # Append the conda-forge zoneinfo to the end
   sed -i.bak "s@zoneinfo'@zoneinfo:$PREFIX/share/tzinfo'@g" sysconfigfile
   # Remove osx sysroot as it depends on the build machine
-  sed -i.bak "s@-isysroot @@g" sysconfigfile
-  sed -i.bak "s@$CONDA_BUILD_SYSROOT @@g" sysconfigfile
+  # sed -i.bak "s@-isysroot ${CONDA_BUILD_SYSROOT}@@g" sysconfigfile
```

Comment:
Why?

Reply:
Because: https://github.com/conda-forge/python-feedstock/pull/448/files#diff-e0638ff31f64a715c0ed3faa90c7b9f61ef3ff28d4a622e89e31923e585394bbR387-R388

FWIW, the method did not work depending on the delineation / line wrapping that python was applying to sysconfig.py.

conda-build has its own way of doing replacements based on regexs now that I want to use instead of this stuff though and I will work later (if I can) on using that new method. Currently I used it in our Qt5 recipe and it works well there.

Can you be more detailed with your question though, I would like some more context if possible.

Reply:
> FWIW, the method did not work depending on the delineation / line wrapping that python was applying to sysconfig.py.

Yes, that's why we had the two lines. I think you might have been looking at a commit where we didn't fix the line wrapping.

Anyways, I didn't realize there were substitutions done using `_FLAGS_REPLACE`. Do you think we should move those here to use `sed` or move these `sed` commands there to use `replace-word-pairs.py`. FWIW, I like the `sed` commands more because I think it's slightly more readable.